react.html

```diff
@@ -65,10 +65,19 @@
 	  	  	strikes: 0
 	  	  };
   	  },
+  	  timerTick: function() {
+  	  	this.setState({
+  	  		strikes: this.state.strikes + 100
+  	  	});
+  	  },
+  	  componentDidMount: function() {
+  	  	setInterval(this.timerTick, 1000);
```

Comment:
When this component unmounts, the timer will keep running every second, which may cause errors at the worst, or waste resources at the best. In this situation, you should save the timer id (maybe into state), and define a componentWillUnmount method that calls clearInterval to stop the timer.

You can see an example of this on https://facebook.github.io/react/ under "A Stateful Component". I'll paste it here for posterity:

```jsx
var Timer = React.createClass({
  getInitialState: function() {
    return {secondsElapsed: 0};
  },
  tick: function() {
    this.setState({secondsElapsed: this.state.secondsElapsed + 1});
  },
  componentDidMount: function() {
    this.interval = setInterval(this.tick, 1000);
  },
  componentWillUnmount: function() {
    clearInterval(this.interval);
  },
  render: function() {
    return (
      <div>Seconds Elapsed: {this.state.secondsElapsed}</div>
    );
  }
});
```